src/plugins/WebIndexPlugin.ts

```diff
@@ -86,7 +87,7 @@ $bundles
 
         let jsTags = this.opts.emitBundles
             ? this.opts.emitBundles(bundlePaths)
-            : bundlePaths.map(bundle => `<script ${this.opts.async ? 'async' : ''} type="text/javascript" src="${bundle}"></script>`).join('\n');
+            : bundlePaths.map(bundle => `<script ${this.opts.async ? this.opts.async : ''} ${this.opts.scriptAttributes ? this.opts.scriptAttributes : ''} type="text/javascript" src="${bundle}"></script>`).join('\n');
```

Comment:
I believe it was `${this.opts.async ? 'async' : ''}`
sorry man for being picky ;-) But I don't want to have any breaking changes.

Reply:
@nchanged no prob. I'm travelling these days. I missed that typo. Fixing and committing! Sorry and Thanks